sites/all/themes/orwell/sass/component/_article.scss

```diff
@@ -1,6 +1,7 @@
 @import 'bootstrap/bootstrap';
 
-$teaser-bg-default: #393939;
+$teaser-height--default: 370px;
```

Comment:
Do we need a modifier for default? :p

Reply:
Well, if you put it that way... No...

The whole concept of BEM for variable names is new to me.

There'll be a few other places I'll need to fix this.